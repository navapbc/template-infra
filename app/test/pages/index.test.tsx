// test/pages/index.test.js

import { render, screen } from '@testing-library/react'
import { axe } from 'jest-axe'
import Home from '@pages/index'

describe('Home', () => {
  it('should render the heading', () => {
    render(<Home />)

    const heading = screen.getByText(/Next.js Template!/i)

    expect(heading).toBeInTheDocument()
    expect(heading).toMatchSnapshot()
  })

  it('should pass accessibility scan', async () => {
    const { container } = render(<Home />)
    const results = await axe(container)

    expect(results).toHaveNoViolations()
  })
})
