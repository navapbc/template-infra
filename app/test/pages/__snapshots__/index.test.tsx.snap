// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render the heading 1`] = `
<a
  href="https://github.com/navapbc/template-application-nextjs"
>
  Next.js Template!
</a>
`;
